packages/components/src/Workflow/subcomponents/Wrapper/Wrapper.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { PropsWithChildren, HTMLAttributes } from "react"
+import classnames from "classnames"
+import { OverrideClassName } from "~types/OverrideClassName"
+import styles from "./Wrapper.module.scss"
+export type WorkflowWrapperProps = PropsWithChildren<
+  OverrideClassName<HTMLAttributes<HTMLDivElement>>
+>
```

Comment:
```suggestion
export type WorkflowWrapperProps = OverrideClassName<HTMLAttributes<HTMLDivElement>> & {
  children: React.ReactNode
}
```

I'd probably why away from helper types as they can change between React versions. Better to be explicit (it's why we got rid of React.FC).

Reply:
Fair call, better to be explicit and safe